### ISSUE TITLE
block also www.* variants of porn sites

### ISSUE THREAD

**Issue by Shujito** [Clefspeare13/pornhosts#12]
e.g. `xvideos.com` is blocked, but `www.xvideos.com` is not

it'd be good to include these as well

**Comment by Clefspeare13** [comment]
Have you been able to access `www.xvideos.com` but not `xvideos.com`? I thought it wouldn't make a difference, but if it does for sure I can add these variations.

**Comment by IDKwhattoputhere** [comment]
From my own experience it makes a difference when blocking with the hosts file (under Windows and Android at least). All subdomains have to be blocked seperately including `www.`
This article has some good information on this: https://www.online-tech-tips.com/computer-tips/edit-windows-hosts-file-to-block-redirect-websites/

> For example, **it makes a difference if you use www.nytimes.com as opposed to nytimes.com** without the www. Visit the website and **see exactly what the URL is** for the website you want to redirect.

**Comment by gianmaxfactor** [comment]
I confirm that on Windows (at least) it is necessary to add the www prefix.
I am using the latest "Unified hosts + fakenews + gambling + porn + social" and I can navigate to the porn site https://www.amaporn.com because in the hosts file there is just the amaporn.com entry that does not work with regard to the www.amaporn.com url.

**Comment by Clefspeare13** [comment]
Alright, I will be adding www.* versions of all domains in my next update, hopefully by tomorrow evening. Thanks for bringing this to my attention and verifying the issue.

**Comment by Imparium** [comment]
Also you don't need the site.com plus www.site.com
When you put the www. first it also blocks the original site (i.e. without the www subdomain)

**Comment by IDKwhattoputhere** [comment]
@Imparium That depends. True for sites that redirect from `site.com` to `www.site.com`. Some sites don't or always use `site.com` instead of `www.site.com`.
Safest would be to add both `site.com` and `www.site.com`.

**Comment by ColinMaudry** [comment]
I see also a trend of porn sites adding language subdomains that need to be blocked separately (fr, it, de, pt, etc.).

See for instance: https://github.com/Clefspeare13/pornhosts/pull/16/files#diff-23b1a7697c48ef7ddc90f8bf0943ee2d

**Comment by rayneangeles** [comment]
Open

**Comment by primeapple** [comment]
Well, it seems, that the hosts added by @ColinMaudry in PR #16  are not in the host files anymore? Does anyone know where they have been gone?

**Comment by StevenBlack** [comment]
Hello!

You know, if the missing `www.` porn subdomains are a problem, why don't we hear about it?

I don't hear anyone complaining.  The porn hosts files are working.

**Comment by primeapple** [comment]
Hi @StevenBlack .
Sadly you are wrong. Your combined hosts file are the reason I found this. Look at #52 .
For example fr.pornhub.com is not blocked by your hosts file. However it was before, but this was reverted.

**Comment by StevenBlack** [comment]
Um, @primeapple, this issue – see the issue title ☝️– is about `www.` subdomains.  

This issue is also 2-years old.  Nobody has posted a comment in this issue in 15 months.

Maybe open a separate issue for country-prefix domains?  List them, it will get fixed.

**Comment by spirillen** [comment]
Let's have a few corpses on the table.... The requirements for whether you need the `www.` or depends on the server you are visiting. Is the serer (site) setup to redirect the one or the other way.

This means you can't in any way predict this in advance, but you need to test both to validate the need of `www.` or not.  It can just as easily be you need the `www.` and not the domain without the `www.`

But the OP has a point in his argumentation, the answer is just it varies on domain to domain.

Let's take an example

`porno333.com` this site redirects from www to non www
![image](https://user-images.githubusercontent.com/44526987/79703071-a6569f00-82a9-11ea-9bc5-1f79a19db4c1.png)

<details><summary>Logger output</summary>

| | | | | | | |
|:--- |:--- |:--- |:--- |:--- |:--- |:--- |
| +2 | ##[id*="ScriptRoot"] |  | porno333.com |  | dom | `https://porno333.com/` |
| +2 | &action=js_stats& | -- | porno333.com | 1 | image | `https://porno333.com/?mode=async&action=js_stats&rand=1587340431100` |
| +1 |  |  | porno333.com | 1 | image | `https://porno333.com/favicon.ico` |
| +1 | \|\|skkybrought.pro^$3p | -- | porno333.com | 3 | script | `https://skkybrought.pro/v3/a/isv/js/40316?video=video.fp-engine` |
| +1 | \|\|exosrv.com^ | -- | porno333.com | 3 | script | `https://ads.exosrv.com/ads.js` |
| +1 | \|\|adskeeper.co.uk^ | -- | porno333.com | 3 | script | `https://jsc.adskeeper.co.uk/p/o/porno333.com.278829.js?t=12031923` |
| +1 |  |  | behind-the-scene | 0,3 | xhr | `https://255.255.255.255/moz-extension://53cfdd28-130e-46ec-8607-995049ad51d9/?id=152a1813a41.a93%2Chttps%3A%2F%2Fporno333.com%2F&url=https%3A%2F%2Fporno333.com%2F&suspend=true` |
| +1 | \|\|meowpushnot.com^$3p | -- | porno333.com | 3 | script | `https://meowpushnot.com/aas/r45d/vki/1638730/tghr.js` |
| +1 | *$script,domain=porno333.com | -- | porno333.com | 1 | script | `https://porno333.com/kIg1riaq.js` |
| +1 | *$script,domain=porno333.com | -- | porno333.com | 1 | script | `https://porno333.com/FrzUfBI4.js` |
| +1 | \|\|meowpushnot.com^$3p | -- | porno333.com | 3 | script | `https://meowpushnot.com/pn07uscr/f/tr/zavbn/1582325/lib.js` |
| +1 |  |  | behind-the-scene | 0,3 | xhr | `https://255.255.255.255/moz-extension://53cfdd28-130e-46ec-8607-995049ad51d9/?id=10dad22760c.f5c%2Chttps%3A%2F%2Fporno333.com%2F&url=https%3A%2F%2Fporno333.com%2F&top=true&suspend=true` |
| +1 | \|\|skkybrought.pro^$3p | -- | porno333.com | 3 | script | `https://skkybrought.pro/v3/a/isv/js/40316?video=video.fp-engine` |
| +1 | @@\|\|porno333.good-cdn.com/js/main.min.js$script,domain=porno333.com | ++ | porno333.com | 3 | script | `https://porno333.good-cdn.com/js/main.min.js?v=5.0` |
| +1 | \|\|exosrv.com^ | -- | porno333.com | 3 | script | `https://ads.exosrv.com/ads.js` |
| +1 | \|\|meowpushnot.com^$3p | -- | porno333.com | 3 | script | `https://meowpushnot.com/aas/r45d/vki/1638730/tghr.js` |
| +1 | *$script,domain=porno333.com | -- | porno333.com | 1 | script | `https://porno333.com/kIg1riaq.js` |
| +1 | *$script,domain=porno333.com | -- | porno333.com | 1 | script | `https://porno333.com/FrzUfBI4.js` |
| +1 | \|\|meowpushnot.com^$3p | -- | porno333.com | 3 | script | `https://meowpushnot.com/pn07uscr/f/tr/zavbn/1582325/lib.js` |
| +1 |  |  | porno333.com | 3 | css | `https://porno333.good-cdn.com/styles/styles.min.css` |
| +1 | no-remote-fonts: * true | -- | porno333.com | 1 | inline-font | `https://porno333.com/` |
| +1 |  |  | porno333.com | 1 | doc | `https://porno333.com/` |
| +1 | no-remote-fonts: * true | -- | `www.porno333.com` | 1 | inline-font | `https://www.porno333.com/` |
| +0 |  |  | `www.porno333.com` | 1 | doc | `https://www.porno333.com/` |
</details>


`asiansgoporn.com` serves both
![image](https://user-images.githubusercontent.com/44526987/79703118-ffbece00-82a9-11ea-98ab-5c236d985918.png)

![image](https://user-images.githubusercontent.com/44526987/79703126-0b11f980-82aa-11ea-9404-1489d9badd5a.png)

<details><summary>Logger output</summary>

| | | | | | | |
|:--- |:--- |:--- |:--- |:--- |:--- |:--- |
| +30 | ##a[title="Back to top"] |  | asiansgoporn.com |  | dom | `https://asiansgoporn.com/` |
| +30 | ###back-to-top |  | asiansgoporn.com |  | dom | `https://asiansgoporn.com/` |
| +29 |  |  | asiansgoporn.com | 1 | image | `https://asiansgoporn.com/wp-content/uploads/2019/06/fav-64-asiangoporn.png` |
| +29 | \|\|wpu.sh^$3p | -- | asiansgoporn.com | 3 | script | `https://sw.wpu.sh/script/main.js?promo=21129&tcid=3158&src=1698053270` |
| +29 | \|\|wpnjs.com^$3p | -- | asiansgoporn.com | 3 | script | `https://js.wpnjs.com/pn.php` |
| +29 | \|\|exosrv.com^ | -- | asiansgoporn.com | 3 | script | `https://a.exosrv.com/ads.js` |
| +29 | \|\|exosrv.com^ | -- | asiansgoporn.com | 3 | script | `https://a.exosrv.com/ads.js` |
| +29 | \|\|exosrv.com^ | -- | asiansgoporn.com | 3 | script | `https://a.exosrv.com/ads.js` |
| +29 | \|\|exosrv.com^ | -- | asiansgoporn.com | 3 | script | `https://a.exosrv.com/nativeads.js` |
| +29 | no-remote-fonts: * true | -- | asiansgoporn.com | 1 | font | `https://asiansgoporn.com/wp-content/themes/retrotube/assets/stylesheets/font-awesome/fonts/fontawesome-webfont.ttf?v=4.7.0` |
| +29 | \|\|exosrv.com^ | -- | asiansgoporn.com | 3 | script | `https://a.exosrv.com/nativeads.js` |
| +29 | no-remote-fonts: * true | -- | asiansgoporn.com | 1 | font | `https://asiansgoporn.com/wp-content/themes/retrotube/assets/stylesheets/font-awesome/fonts/fontawesome-webfont.woff?v=4.7.0` |
| +29 | no-remote-fonts: * true | -- | asiansgoporn.com | 1 | font | `https://asiansgoporn.com/wp-content/themes/retrotube/assets/stylesheets/font-awesome/fonts/fontawesome-webfont.woff2?v=4.7.0` |
| +29 | \|\|exosrv.com^ | -- | asiansgoporn.com | 3 | script | `https://a.exosrv.com/ads.js` |
| +29 | \|\|exosrv.com^ | -- | asiansgoporn.com | 3 | script | `https://a.exosrv.com/popunder1000.js` |
| +29 |  |  | asiansgoporn.com | 1 | script | `https://asiansgoporn.com/wp-content/cache/autoptimize/js/autoptimize_dca47d06ad6a618cb39bef3b2e03dfac.js` |
| +29 | \|\|wpnjs.com^$3p | -- | asiansgoporn.com | 3 | script | `https://js.wpnjs.com/pn.php` |
| +29 | \|\|googleapis.com^$important | -- | asiansgoporn.com | 3 | css | `https://fonts.googleapis.com/css?family=Open+Sans:300,400&subset=cyrillic,greek,latin-ext` |
| +29 | \|\|googletagmanager.com/gtag/$3p | -- | asiansgoporn.com | 3 | script | `https://www.googletagmanager.com/gtag/js?id=UA-143928522-5` |
| +29 | \|\|yadro.ru^ | -- | asiansgoporn.com | 3 | image | `https://counter.yadro.ru/hit?29.6;r;s1000*450*24;uhttps%3A//asiansgoporn.com/;h;0.015889545594361953` |
| +29 |  |  | behind-the-scene | 0,3 | xhr | `https://255.255.255.255/moz-extension://53cfdd28-130e-46ec-8607-995049ad51d9/?id=2d60a1358d.4e58%2Chttps%3A%2F%2Fasiansgoporn.com%2F&url=https%3A%2F%2Fasiansgoporn.com%2F&top=true&suspend=true` |
| +29 | \|\|exosrv.com^ | -- | asiansgoporn.com | 3 | script | `https://a.exosrv.com/nativeads.js` |
| +29 | \|\|exosrv.com^ | -- | asiansgoporn.com | 3 | script | `https://a.exosrv.com/ads.js` |
| +29 |  |  | asiansgoporn.com | 3 | script | `https://cdn.fluidplayer.com/v2/current/fluidplayer.min.js` |
| +29 |  |  | asiansgoporn.com | 3 | css | `https://cdn.fluidplayer.com/v2/current/fluidplayer.min.css` |
| +29 |  |  | asiansgoporn.com | 1 | css | `https://asiansgoporn.com/wp-content/cache/autoptimize/css/autoptimize_d3727c2a9758f4203dc8dab3fc818a0b.css` |
| +29 | \|\|googletagmanager.com/gtag/$3p | -- | asiansgoporn.com | 3 | script | `https://www.googletagmanager.com/gtag/js?id=UA-143928522-5` |
| +29 | no-remote-fonts: * true | -- | asiansgoporn.com | 1 | inline-font | `https://asiansgoporn.com/` |
| +28 |  |  | asiansgoporn.com | 1 | doc | `https://asiansgoporn.com/` |
| +28 | no-remote-fonts: * true | -- | `www.asiansgoporn.com` | 1 | inline-font | `https://www.asiansgoporn.com/` |
| +27 |  |  | `www.asiansgoporn.com` | 1 | doc | `https://www.asiansgoporn.com/` |
| +4 | ###back-to-top |  | asiansgoporn.com |  | dom | `https://asiansgoporn.com/` |
| +4 | ##a[title="Back to top"] |  | asiansgoporn.com |  | dom | `https://asiansgoporn.com/` |
| +4 |  |  | asiansgoporn.com | 1 | image | `https://asiansgoporn.com/wp-content/uploads/2019/06/busty-asian-hoe-slammed-by-cock.jpg` |
| +4 |  |  | asiansgoporn.com | 1 | image | `https://asiansgoporn.com/wp-content/uploads/2019/06/mimi-asuka-fucked-in-a-restaurant-in-public-640x360.jpg` |
| +4 |  |  | asiansgoporn.com | 1 | image | `https://asiansgoporn.com/wp-content/uploads/2019/06/akari-niiyama-fucks-the-man-who-saves-her-640x360.jpg` |
| +4 |  |  | asiansgoporn.com | 1 | image | `https://asiansgoporn.com/wp-content/uploads/2019/06/home-alone-lulu-kinouchi-is-screwed-big-time-640x360.jpg` |
| +4 |  |  | asiansgoporn.com | 1 | image | `https://asiansgoporn.com/wp-content/uploads/2019/06/tiara-kujyo-gets-cum-in-brown-cunt-at-massage-640x360.jpg` |
| +4 |  |  | asiansgoporn.com | 1 | image | `https://asiansgoporn.com/wp-content/uploads/2019/06/satsuki-kirioka-and-azusa-misaki-in-groupsex-640x360.jpg` |
| +4 |  |  | asiansgoporn.com | 1 | image | `https://asiansgoporn.com/wp-content/uploads/2019/06/azusa-misaki-in-uniform-fucked-at-school-640x360.jpg` |
| +4 |  |  | asiansgoporn.com | 1 | image | `https://asiansgoporn.com/wp-content/uploads/2019/06/mirei-oomori-from-wedding-to-crazy-gangbang-640x360.jpg` |
| +4 |  |  | asiansgoporn.com | 1 | image | `https://asiansgoporn.com/wp-content/uploads/2019/06/koi-miyamura-and-boyfriend-fucking-a-lot-640x360.jpg` |
| +4 |  |  | asiansgoporn.com | 1 | image | `https://asiansgoporn.com/wp-content/uploads/2019/06/horny-tsukushi-fucks-her-stepbrothers-cock-640x360.jpg` |
| +4 |  |  | asiansgoporn.com | 1 | image | `https://asiansgoporn.com/wp-content/uploads/2019/06/hibiki-ohtsuki-has-pussy-pleased-and-gives-bj-624x360.jpg` |
| +4 |  |  | asiansgoporn.com | 1 | image | `https://asiansgoporn.com/wp-content/uploads/2019/06/kammy-gets-anal-doggiestyle-and-sucks-cock-good.jpg` |
| +4 |  |  | asiansgoporn.com | 1 | image | `https://asiansgoporn.com/wp-content/uploads/2019/06/yui-nagahara-gets-cum-river-in-hairy-cunt-640x360.jpg` |
| +4 |  |  | asiansgoporn.com | 1 | image | `https://asiansgoporn.com/wp-content/uploads/2019/06/ayumi-iwasa-640x360.jpg` |
| +4 |  |  | asiansgoporn.com | 1 | image | `https://asiansgoporn.com/wp-content/uploads/2019/06/satomi-kirihara-gets-three-dicks-and-toys-640x360.jpg` |
| +4 |  |  | asiansgoporn.com | 1 | image | `https://asiansgoporn.com/wp-content/uploads/2019/06/miku-tamaru-gets-fucked-by-her-stepbrother-640x360.jpg` |
| +4 |  |  | asiansgoporn.com | 1 | image | `https://asiansgoporn.com/wp-content/uploads/2019/06/tsukushi-competing-with-babe-for-her-brother-640x360.jpg` |
| +4 |  |  | asiansgoporn.com | 1 | image | `https://asiansgoporn.com/wp-content/uploads/2019/06/maid-nene-azami-satisfying-wild-fantasies-640x360.jpg` |
| +4 |  |  | asiansgoporn.com | 1 | image | `https://asiansgoporn.com/wp-content/uploads/2019/06/satsuki-kirioka-gets-three-cocks-in-pussy-640x360.jpg` |
| +4 |  |  | asiansgoporn.com | 1 | image | `https://asiansgoporn.com/wp-content/uploads/2019/06/shiori-himemiya-gets-facial-after-threesome-640x360.jpg` |
| +4 |  |  | asiansgoporn.com | 1 | image | `https://asiansgoporn.com/wp-content/uploads/2019/06/small-tit-asian-gives-titjob-outdoors.jpg` |
| +4 |  |  | asiansgoporn.com | 1 | image | `https://asiansgoporn.com/wp-content/uploads/2019/06/hottest-nice-ass-fucking-ever.jpg` |
| +4 |  |  | asiansgoporn.com | 1 | image | `https://asiansgoporn.com/wp-content/uploads/2019/06/hottest-amateur-busty-fuck.jpg` |
| +4 |  |  | asiansgoporn.com | 1 | image | `https://asiansgoporn.com/wp-content/uploads/2019/06/hottest-group-filipino-sex-ever.jpg` |
| +4 |  |  | asiansgoporn.com | 1 | image | `https://asiansgoporn.com/wp-content/uploads/2019/06/sexy-asian-amateur-fucks-like-crazy.jpg` |
| +4 |  |  | asiansgoporn.com | 1 | image | `https://asiansgoporn.com/wp-content/uploads/2019/06/hot-karate-pussy-fucking-action.jpg` |
| +4 |  |  | asiansgoporn.com | 1 | image | `https://asiansgoporn.com/wp-content/uploads/2019/06/see-hibiki-ohtsuki-get-double-penetrated-now-624x360.jpg` |
| +4 |  |  | asiansgoporn.com | 1 | image | `https://asiansgoporn.com/wp-content/uploads/2019/06/hibiki-ohtsuki-gives-blowjob-and-gets-facial-624x360.jpg` |
| +4 |  |  | asiansgoporn.com | 1 | image | `https://asiansgoporn.com/wp-content/uploads/2019/06/hibiki-ohtsukis-pussy-eaten-out-and-then-69-624x360.jpg` |
| +4 |  |  | asiansgoporn.com | 1 | image | `https://asiansgoporn.com/wp-content/uploads/2019/06/aya-sakuraba-takes-turns-fucking-and-sucking-624x360.jpg` |
| +4 |  |  | asiansgoporn.com | 1 | image | `https://asiansgoporn.com/wp-content/uploads/2019/06/oiled-up-aya-sakuraba-plays-with-her-vibe-624x360.jpg` |
| +4 |  |  | asiansgoporn.com | 1 | image | `https://asiansgoporn.com/wp-content/uploads/2019/06/aya-sakuraba-has-group-sex-with-blowjobs-too-624x360.jpg` |
| +4 |  |  | asiansgoporn.com | 1 | image | `https://asiansgoporn.com/wp-content/uploads/2019/06/lucy-lee-gets-deep-anal-and-hard-fucking-time.jpg` |
| +4 |  |  | asiansgoporn.com | 1 | image | `https://asiansgoporn.com/wp-content/uploads/2019/06/lucy-lee-fucked-hard-in-pussy-and-tight-ass.jpg` |
| +4 |  |  | asiansgoporn.com | 1 | image | `https://asiansgoporn.com/wp-content/uploads/2019/06/big-tits-mika-kani-gets-hard-anal-and-wild-sex.jpg` |
| +4 |  |  | asiansgoporn.com | 1 | image | `https://asiansgoporn.com/wp-content/uploads/2019/06/jayna-oso-is-bent-over-doggiestyle-with-anal.jpg` |
| +4 |  |  | asiansgoporn.com | 1 | image | `https://asiansgoporn.com/wp-content/uploads/2019/06/kat-gets-double-teamed-in-brutal-threesome.jpg` |
| +4 |  |  | asiansgoporn.com | 1 | image | `https://asiansgoporn.com/wp-content/uploads/2019/06/mika-tan-sucks-cock-and-gets-hard-anal-sex.jpg` |
| +4 |  |  | asiansgoporn.com | 1 | image | `https://asiansgoporn.com/wp-content/uploads/2019/06/shiori-himemiya-gets-toys-in-oiled-beaver-640x360.jpg` |
| +4 |  |  | asiansgoporn.com | 1 | image | `https://asiansgoporn.com/wp-content/uploads/2019/06/runna-sakai-fucked-big-time-in-threesome-640x360.jpg` |
| +4 |  |  | asiansgoporn.com | 1 | image | `https://asiansgoporn.com/wp-content/uploads/2019/06/rena-gets-a-long-strong-fucking-in-hairy-slit-640x360.jpg` |
| +4 |  |  | asiansgoporn.com | 1 | image | `https://asiansgoporn.com/wp-content/uploads/2019/08/asian-slut-fcuked-right-in-subway-640x360.jpg` |
| +4 |  |  | asiansgoporn.com | 1 | image | `https://asiansgoporn.com/wp-content/uploads/2020/02/coronavirus-porn-babe-from-asia.jpg` |
| +4 |  |  | asiansgoporn.com | 1 | image | `https://asiansgoporn.com/wp-content/uploads/2019/06/fav-64-asiangoporn.png` |
| +4 | \|\|wpu.sh^$3p | -- | asiansgoporn.com | 3 | script | `https://sw.wpu.sh/script/main.js?promo=21129&tcid=3158&src=1698053270` |
| +4 | \|\|wpnjs.com^$3p | -- | asiansgoporn.com | 3 | script | `https://js.wpnjs.com/pn.php` |
| +4 | \|\|exosrv.com^ | -- | asiansgoporn.com | 3 | script | `https://a.exosrv.com/ads.js` |
| +4 | \|\|exosrv.com^ | -- | asiansgoporn.com | 3 | script | `https://a.exosrv.com/ads.js` |
| +4 | \|\|exosrv.com^ | -- | asiansgoporn.com | 3 | script | `https://a.exosrv.com/ads.js` |
| +4 | \|\|exosrv.com^ | -- | asiansgoporn.com | 3 | script | `https://a.exosrv.com/nativeads.js` |
| +4 | no-remote-fonts: * true | -- | asiansgoporn.com | 1 | font | `https://asiansgoporn.com/wp-content/themes/retrotube/assets/stylesheets/font-awesome/fonts/fontawesome-webfont.ttf?v=4.7.0` |
| +4 | \|\|exosrv.com^ | -- | asiansgoporn.com | 3 | script | `https://a.exosrv.com/nativeads.js` |
| +3 | no-remote-fonts: * true | -- | asiansgoporn.com | 1 | font | `https://asiansgoporn.com/wp-content/themes/retrotube/assets/stylesheets/font-awesome/fonts/fontawesome-webfont.woff?v=4.7.0` |
| +3 | no-remote-fonts: * true | -- | asiansgoporn.com | 1 | font | `https://asiansgoporn.com/wp-content/themes/retrotube/assets/stylesheets/font-awesome/fonts/fontawesome-webfont.woff2?v=4.7.0` |
| +3 | \|\|exosrv.com^ | -- | asiansgoporn.com | 3 | script | `https://a.exosrv.com/ads.js` |
| +3 | \|\|exosrv.com^ | -- | asiansgoporn.com | 3 | script | `https://a.exosrv.com/popunder1000.js` |
| +3 | \|\|googleapis.com^$important | -- | asiansgoporn.com | 3 | css | `https://fonts.googleapis.com/css?family=Open+Sans:300,400&subset=cyrillic,greek,latin-ext` |
| +2 | \|\|googletagmanager.com/gtag/$3p | -- | asiansgoporn.com | 3 | script | `https://www.googletagmanager.com/gtag/js?id=UA-143928522-5` |
| +2 | \|\|yadro.ru^ | -- | asiansgoporn.com | 3 | image | `https://counter.yadro.ru/hit?29.6;r;s1000*450*24;uhttps%3A//asiansgoporn.com/;h;0.3244819733877843` |
| +2 |  |  | asiansgoporn.com | 1 | script | `https://asiansgoporn.com/wp-content/cache/autoptimize/js/autoptimize_dca47d06ad6a618cb39bef3b2e03dfac.js` |
| +2 | \|\|wpnjs.com^$3p | -- | asiansgoporn.com | 3 | script | `https://js.wpnjs.com/pn.php` |
| +2 |  |  | asiansgoporn.com | 1 | image | `https://asiansgoporn.com/wp-content/themes/retrotube/assets/img/px.gif` |
| +2 | \|\|exosrv.com^ | -- | asiansgoporn.com | 3 | script | `https://a.exosrv.com/nativeads.js` |
| +2 | \|\|exosrv.com^ | -- | asiansgoporn.com | 3 | script | `https://a.exosrv.com/ads.js` |
| +2 |  |  | asiansgoporn.com | 1 | image | `https://asiansgoporn.com/wp-content/uploads/2019/06/logo-asiangoporn2.png` |
| +2 |  |  | asiansgoporn.com | 3 | script | `https://cdn.fluidplayer.com/v2/current/fluidplayer.min.js` |
| +2 |  |  | asiansgoporn.com | 3 | css | `https://cdn.fluidplayer.com/v2/current/fluidplayer.min.css` |
| +2 |  |  | asiansgoporn.com | 1 | css | `https://asiansgoporn.com/wp-content/cache/autoptimize/css/autoptimize_d3727c2a9758f4203dc8dab3fc818a0b.css` |
| +2 |  |  | behind-the-scene | 0,3 | xhr | `https://255.255.255.255/moz-extension://53cfdd28-130e-46ec-8607-995049ad51d9/?id=b6253aa2af.3058%2Chttps%3A%2F%2Fasiansgoporn.com%2F&url=https%3A%2F%2Fasiansgoporn.com%2F&top=true&suspend=true` |
| +2 | \|\|googletagmanager.com/gtag/$3p | -- | asiansgoporn.com | 3 | script | `https://www.googletagmanager.com/gtag/js?id=UA-143928522-5` |
| +2 | no-remote-fonts: * true | -- | asiansgoporn.com | 1 | inline-font | `https://asiansgoporn.com/` |
| +0 |  |  | asiansgoporn.com | 1 | doc | `https://asiansgoporn.com/` |
</details>

As you can see, there a no either of but deffently a :+1: to OP @Shujito for bringing this to attention.

I leave it up to @Clefspeare13 to close this issue or pin it as remember roll

**Comment by spirillen** [comment]
This domain for example serves both `www.xxxt.pro` and `xxxt.pro`

![image](https://user-images.githubusercontent.com/44526987/79703330-6690b700-82ab-11ea-8bf4-c82630fe04a9.png)
![image](https://user-images.githubusercontent.com/44526987/79703338-75776980-82ab-11ea-81a8-326a64f24ae6.png)
